libs/hwes/event.go

```diff
@@ -48,7 +56,35 @@ type metadata struct {
 	TraceParent string `json:"trace_parent"`
 	// The Timestamp represents the time when the event was created. Using the built-in eventstoreDB timestamp is
 	// discouraged.
-	Timestamp time.Time `json:"timestamp"`
+	Timestamp   time.Time `json:"timestamp"`
+	DataIsProto bool      `json:"data_is_proto"`
+}
+
+// MessageToEventName reflects the passed EventType message extension of the passed message.
+// When the EventType is nil, the current default EventType of proto/libs/events/v1 gets used.
```

Comment:
```suggestion
// When the EventType is nil, the current default EventType of proto/libs/events/v1 gets used.
```

instead, drop the second parameter and suffix the function with v1 or alike